tests/unit/select_test.coffee

```diff
@@ -344,3 +344,22 @@ test 'optionValuePath with nested valuePath', ->
 
   equal obj2, select.get('selection'),
     'The right selection is retrieved'
+
+test 'shouldEnsureVisible controls whether to ensure visibility', ->
+  expect 2
+
+  select = @subject
+    content: ['foo']
+    ensureVisible: ->
+      ok yes, 'ensureVisible is called if shouldEnsureVisible is true'
```

Comment:
I have to stub the content of `ensureVisible` otherwise the test will throw exception. This is the most easy-to-understand way IMO